frontend/packages/console-shared/src/components/formik-fields/item-selector-field/SelectorCard.tsx

```diff
@@ -18,22 +17,15 @@ const SelectorCard: React.FC<SelectorCardProps> = ({
   title,
   iconUrl,
   name,
-  displayName,
   selected,
   recommended = false,
   onChange,
 }) => {
   const classes = classNames('odc-selector-card', { 'is-selected': selected });
   return (
-    <Card
-      component="button"
-      type="button"
-      aria-label={title}
-      className={classes}
-      onClick={() => onChange(name)}
-    >
+    <Card component="button" type="button" className={classes} onClick={() => onChange(name)}>
       <CardTitle>
-        <img className="odc-selector-card__icon" src={iconUrl} alt={displayName ?? title} />
+        <img className="odc-selector-card__icon" src={iconUrl} alt={title} aria-hidden />
```

Comment:
If the image is just decorative and the title is diplayed elsewhere on the card, this can just be `alt=""` (then you wouldn't need `aria-hidden`).

Reply:
Thanks for the review. :)
The last time I added alt="" the reviewer said I should avoid this because it is also shown if the image is not loaded.. Both true. ;)

Reply:
I'm confused: what is shown? We're setting `alt` to an empty value, right? My understanding is that `alt=""` is preferred for decorative images that can be skipped by a screen reader.

Reply:
https://www.w3.org/WAI/tutorials/images/decorative/

> Decorative images don’t add information to the content of a page. For example, the information provided by the image might already be given using adjacent text, or the image might be included to make the website more visually attractive.
> 
> In these cases, a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers. Text values for these types of images would add audible clutter to screen reader output or could distract users if the topic is different from that in adjacent text. Leaving out the alt attribute is also not an option because when it is not provided, some screen readers will announce the file name of the image instead.

Reply:
Thanks @spadgett. Rebased and just changed this to `alt=""`